pydantic/plugin/_loader.py

```diff
@@ -1,16 +1,8 @@
 from __future__ import annotations
 
-import sys
+import importlib.metadata as importlib_metadata
 import warnings
-from typing import TYPE_CHECKING, Iterable
-
-from typing_extensions import Final
-
-if sys.version_info >= (3, 8):
-    import importlib.metadata as importlib_metadata
-else:
-    import importlib_metadata
-
+from typing import TYPE_CHECKING, Final, Iterable
```

Comment:
Why the `Final` here?

Reply:
The `Final` annotation is effective from [PEP591](https://peps.python.org/pep-0591/#semantics-and-examples), thus we can adopt this starting from version `3.8`